apps/builder/app/builder/features/style-panel/shared/css-value-input/unit-select-options.ts

```diff
@@ -26,7 +26,17 @@ const preferedSorting = [
   ...units.time,
 ];
 
-const visibleLengthUnits = ["px", "em", "rem", "dvw", "dvh"] as const;
+const initialLengthUnits = [
+  "px",
+  "%",
+  "em",
+  "rem",
+  "ch",
+  "svw",
+  "svh",
+  "lvw",
+  "lvh",
```

Comment:
We really need descriptions here caz I have now idea what these for 🤷

Reply:
valid point, separate pr, need to come up with very short but very clear descriptions, that won't be easy

Reply:
webflow made a great video on this https://university.webflow.com/lesson/small-large-and-dynamic-viewport-units?topics=layout-design